website/docs/js-delivery-preview-launch.md

```diff
@@ -9,7 +9,7 @@ Whenever you make changes to experiments in Mojito, you need to build/publish yo
 
 1.  (Optional) Customise your Mojito container name in [`repo/config.js`](https://github.com/mint-metrics/mojito-js-delivery/blob/master/config.example.js)
 2.  If you haven't done so already, install the necessary NPM packages: `npm install`
```

Comment:
`npm install` is changed to `npm ci` in the intro.md file. Should that be reflected here as well?

Reply:
@allmywant - this is a good question. I just assumed `npm ci` is the right way to install it based on what we have in Bitbucket Pipelines. Should we be using `npm ci` or `npm install`?

Reply:
@kingo55 
We use `npm ci` in Bitbucket Pipelines to ensue npm installs dependencies with the versions to be exactly same as the ones specified in package-lock.json. For local installs, both `npm install` and `npm ci` work as expected. Let's use `npm install` because it is more commonly used.
In the next version, I will change the dependency versions to exact numbers, e.g. from `"clean-css": "^5.1.2"` to `"clean-css": "5.1.2"`. Then we can use `npm install` for Bitbucket Pipelines and local environments.